gnome-extra/gnome-commander/gnome-commander-1.10.0.ebuild

```diff
@@ -52,6 +51,13 @@ src_configure() {
 		$(use_with unique)
 }
 
+pkg_setup() {
```

Comment:
Use `pkg_pretend` so that user knows early. It sucks when you run batch of updates that are supposed to run for hours just to have them die midway.